gitlab-ci/php-ci.md

```diff
@@ -0,0 +1,20 @@
+# Configuration
+
+Create a `.gitlab-ci.yml` file inside your project root directory and copy the below configuration.
+
+```yaml
+stages:
+  - lint  
+
+lint:
+  stage: lint
+  script: 
+    - echo "Linting code..."
+    - composer install
+    - composer run lint
+    - echo "Static code analysis..."
```

Comment:
This should be a separate stage actually

Reply:
I tried this, but the vendor modules that are installed in the previous stage are not available in the next one. Installing them in the next stage with `composer install`, I thought would be overkill and did not do it.